apps/ideas/src/components/modules/IdeaGeneratorButton/IdeaGeneratorButton.js

```diff
@@ -54,7 +65,7 @@ export default function IdeaGeneratorButton(props) {
         openModal={openModal}
         closeModal={closeModal}
       ></Modal>
-      <div style={{ width: "100%", height: "100%" }} onClick={openIdeaModal}>
+      <div style={{ width: '100%', height: '100%' }} onClick={openIdeaModal}>
```

Comment:
not required for this ticket, but it would be nice to refactor this styling going forward.